deps/README.md

```diff
@@ -0,0 +1,3 @@
+# Deps
+
+Deps contains dependencies that for some reason or another weren't importable into vendor/.
```

Comment:
I'll just delete this dir. It's no longer used.

google/protobuf was not importable into vendor because it needs to live in the directory corresponding to it's proto path. 

But I stopped using it.